cheat/cheatsheets/git

```diff
@@ -17,9 +17,16 @@ git commit -m "Your commit message"
 # To edit previous commit message
 git commit --amend
 
+# Git commit in the past
+git commit --date="`date --date='2 day ago'`"
```

Comment:
Good to know, thanks. I tried looking through the log of Git’s `date.c`, where approxidate (thanks for the name btw) reportedly lives, but it’s not clear to me when support for this would have been added – it looks like a lot of gradual improvements to me, not one point in time when git jumped from “must be exactly *that* format” to “lol I’ll eat anything”.

I suggest the following version:
```sh
# git commit in the past
git commit --date="`date --date='2 days ago'`"
# more recent versions of Git also support --date="2 days ago" directly
```